src/models/RegistrationSchema.js

```diff
@@ -34,6 +34,10 @@ export default BaseSchema.Model.extend({
     BaseModel.apply(this, arguments);
   },
 
+  initialize: function () {
+    this.settings = this.settings || this.options.settings;
```

Comment:
why need this change?

Reply:
For RegistrationSchema tests. I wanted to pass a settings object, see [this change](https://github.com/okta/okta-signin-widget/pull/1747/commits/84a33bf2495d9fea9af5afc7dbcd481c8e2322eb#diff-fc462337c5af938bc946ddbf325400ae74fb5081cdec6314a3a092a810b90869R6-R14)

Reply:
Eh I can do without it, I removed it [here](https://github.com/okta/okta-signin-widget/pull/1747/commits/8af51133829b6b27449ec0febdd97d9cd6e0d329)